db/utility.js

```diff
@@ -0,0 +1,52 @@
+import { dbQuery } from "./connection.js"
+const queryAllItem = async()=>{
+    let itemTransResult = await dbQuery('SELECT * FROM `Item_Trans` WHERE `lang`=\'zh\'')
+    let itemResult = await dbQuery('SELECT * FROM `Item`')
+    let commentResult = await dbQuery('SELECT * FROM `Comment`')
+    let itemCommentResult = await dbQuery('SELECT * FROM `Item_Comment`')
+    let result = []
+    let itemList = []
+    let itemCommentObj = {}
+    itemTransResult.forEach((e)=>{
+        itemList = [...itemList, e.itemId]
+        itemCommentObj[e.itemId] = []
+    })
+    // handel comments
```

Comment:
instead of writing a comment just move the code below to a new function called `handleComments()` and invoke this function here. This technique is a part of clean code guidelines by Robert Martin https://gist.github.com/wojteklu/73c6914cc446146b8b533c0988cf8d29

Reply:
same with the code below

Reply:
okay, I fixed it.